apps/src/applab/appStorage.js

```diff
@@ -38,6 +38,10 @@ var handleGetKeyValue = function(onSuccess, onError) {
   if (this.readyState !== 4) {
```

Comment:
It also feels a little weird to bind this to be the xhr, where in most cases in this file I would expect this to refer to AppStorage. I wonder if it would it make more sense to pass the xhr into handleGetKeyValue instead?

Also, sorry for commenting on things that aren't actually a part of your PR